FutureFeatures.md

```diff
@@ -425,3 +415,46 @@ of WebAssembly in browsers:
   would enable Web apps to perform their own (["layer 1"](BinaryEncoding.md))
   custom compression (on top of the spec-defined binary format, under generic
   HTTP `Content-Encoding` compression).
+
+## Multiple Tables and Memories
+
+In the MVP, there can only be (at most one) default table/memory and there are
```

Comment:
Fixed, but re-added a bit at the end for "and there are only operators for accessing the default table and memory".